recipes/conversation_view/feed_pill/feed_item_pill.stories.js

```diff
@@ -62,42 +64,28 @@ const DefaultTemplate = (args, { argTypes }) => createTemplateFromVueFile(
   DtRecipeFeedItemPillDefaultTemplate,
 );
 
+const VariantsTemplate = (args, { argTypes }) => createTemplateFromVueFile(
+  args,
+  argTypes,
+  DtRecipeFeedItemPillVariantsTemplate,
+);
+
 // Stories
 export const Default = {
   render: DefaultTemplate,
-  parameters: { options: { showPanel: false }, a11y: { disable: true }, controls: { disable: true } },
-};
-
-export const DefaultToggledVariant = {
-  render: DefaultTemplate,
-  parameters: Default.parameters,
-  args: {
-    defaultToggled: true,
-  },
-};
-
-export const DefaultToggledVariantNonToggleable = {
-  render: DefaultTemplate,
-  parameters: Default.parameters,
-  args: {
-    toggleable: false,
-    defaultToggled: true,
-  },
-};
-
-export const AiBorderGradient = {
-  render: DefaultTemplate,
-  parameters: Default.parameters,
-  args: {
-    borderColor: 'ai',
+  parameters: {
+    options: { showPanel: false },
+    a11y: { disable: true },
+    percy: {
+      args: { seed: 'seed' },
+    },
   },
 };
 
-export const NoToggleVariant = {
-  render: DefaultTemplate,
-  parameters: Default.parameters,
-  args: {
-    toggleable: false,
-    borderColor: 'critical',
+export const Variants = {
+  render: VariantsTemplate,
+  parameters: {
+    ...Default.parameters,
+    controls: { disable: true },
```

Comment:
Disable the panel and the controls on variants story.

```suggestion
   parameters: { 
     options: { showPanel: false }, 
     a11y: { disable: true }, 
     controls: { disable: true },
```